script/licenses.py

```diff
@@ -201,7 +201,6 @@ def from_dict(cls, data: PackageMetadata) -> PackageDefinition:
     "pybbox",  # https://github.com/HydrelioxGitHub/pybbox/pull/5
     "pysabnzbd",  # https://github.com/jeradM/pysabnzbd/pull/6
     "pyvera",  # https://github.com/maximvelichko/pyvera/pull/164
-    "repoze.lru",
```

Comment:
I assume it was a third-level transient requirement, and the corresponding second-level requirement was recently updated to remove the dependency